policy/modules/contrib/nbdkit.fc

```diff
@@ -0,0 +1,5 @@
+HOME_DIR/tmp(/.*)?        gen_context(system_u:object_r:nbdkit_home_t,s0)
```

Comment:
This will clash with other files created in HOME_DIR/tmp, we currently have <<none> for all files. Probably better to adjust for particular filenames or remove completely.

Reply:
Thanks, I understand. 

It was based on this path: 
$HOME/tmp/known-hosts

Should I label known-hosts dir?

Reply:
That should work. If such an entry really is required - it applies only when restorecon is run and the files have incorrect label.